NEWS.md

```diff
@@ -4,6 +4,8 @@ User-visible changes worth mentioning.
 
 ## master
 
+## 4.0.0
+
 - [#834] Fix AssetNotPrecompiled error with Sprockets 4
```

Comment:
There are some changes worth mentioning between the latest release and today's master:

- [#847] Specify Null option to timestamps
- [#843] Revert "Fix validation error messages"
- [#847] Specify Null option to timestamps

Cam you please add them? Thanks!